pages/with-app-layout/integ.page.tsx

```diff
@@ -43,7 +43,14 @@ export default function Page() {
     }
   }, [paletteWidgetsLoading]);
 
-  const onChange = ({ items, addedItem, removedItem }: BoardProps.ItemsChangeDetail<ItemData>) => {
+  const onChange = ({
+    items,
+    addedItem,
+    removedItem,
+    movedItem,
+    resizedItem,
+  }: BoardProps.ItemsChangeDetail<ItemData>) => {
+    console.log({ addedItem, removedItem, movedItem, resizedItem });
```

Comment:
debugging leftover

Reply:
Not a leftover, I think it is fine to keep this for easy testing on PR previews

Reply:
Got it. If you want to merge this line I'd add a code comment to prevent this from being removed by someone else.